packages/components/src/internal/app/constants.ts

```diff
@@ -27,6 +27,7 @@ export const WORKFLOW_HOME_HREF = AppURL.create(WORKFLOW_KEY)
     .addParam('mine.sort', 'DueDate')
     .addParam('active.sort', 'DueDate');
 export const NEW_FREEZER_DESIGN_HREF = AppURL.create(FREEZERS_KEY, 'new');
+export const MANAGE_STORAGE_UNITS_HREF = AppURL.create(BOXES_KEY, 'types', 'update');
```

Comment:
Should this be moved to @labkey/freezermanager ?

Reply:
Good thought.  I think I'll leave this here for now, though.  The `NEW_FREEZER_DESIGN_HREF` needs to be here and it could be the MANAGE_STORAGE one would need to be here as well (originally that link was destined for the mega-menu, and it might go back there, depending on approaches to reorganizing that menu.